### PR TITLE
SN-3 Set up axios

### PR DESCRIPTION
**Pull request by popovbojana**

### CODE REVIEW

File: src/services/UserService.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import axios from 'src/config/axios/axios';
+
+export const getUser = async (id: string) => {
+  const response = await axios.get('/users/' + id);
```
Comment (vkotarlic) [inline src/services/UserService.ts]: Please use template literals for concatenating strings

File: src/services/UserService.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import axios from 'src/config/axios/axios';
+
+export const getUser = async (id: string) => {
```
Comment (vkotarlic) [inline src/services/UserService.ts]: Type should be number

File: src/services/UserService.ts (hunk 1 above)
Comment (vkotarlic) [inline src/services/UserService.ts]: No need for variable this should be `return response = await axios.get('/users/' + id);`

**Review by vkotarlic — CHANGES_REQUESTED** [review]
Please resolve conflicts

File: src/services/UserService.ts (hunk 1 above)
Comment (popovbojana) [reply src/services/UserService.ts]: Is this okay `return await axios.get(`/users/${id}`);`?

**Review by vkotarlic — APPROVED** [review]

**Pull request merged**